src/encoded/static/components/item-pages/ExperimentSetView.js

```diff
@@ -312,7 +312,7 @@ export class HiGlassAdjustableWidthRow extends React.PureComponent {
             leftPanelDefaultCollapsed, leftPanelCollapseHeight, leftPanelCollapseWidth, higlassItem } = this.props;
 
         // Don't render the HiGlass view if it isn't mounted yet or there is nothing to display.
-        if (!files || !mounted || !higlassItem) {
+        if (!mounted || !(higlassItem || files) ) {
```

Comment:
Change to:
`if (!mounted || !higlassItem || !object.itemUtil.atId(higlassItem)) {`

We could probably remove `files` from like `propTypes` as well and any other references/mentions of it in this component

Reply:
Then again there's also `collapsedToolTipContent()` that just noticed...

Thinking we could remove the func `collapsedToolTipContent()` entirely and then in render method, change `data-html data-place="right" data-tip={this.collapsedToolTipContent()}>` to be `... data-tip="Open HiGlass Visualization for file(s)">` :-P